public_fips_dropin_environments/python3_base/fit.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
```

Comment:
@eric-s-s 
FYI.
We copy sh from the build image to `/bin/sh` 
So i needed to define shebang this way, different from all other envs else: `#!/usr/bin/env sh` (probably makes sense to update them as well)